packages/Alert/src/index.tsx

```diff
@@ -44,13 +51,41 @@ const AlertComponent = forwardRef<'div', AlertProps>(
     const defaultVariantIcon = variant === 'beige' ? 'default' : variant
 
     const content = Children.toArray(children).map((child: React.ReactElement) => {
-      // Add variant to Title to show the correct icon
-      if (child.type === Title) {
-        return cloneElement(child, { variant })
-      }
+      if (child.type === Title) return cloneElement(child, { variant: size })
+
       return child
     })
 
+    // Handle clone actions recursively in case of multiple buttons
+    const cloneActions = (child: React.ReactElement<AlertProps>): CloneActionsReturns => {
+      if (child.type === AlertPrimaryAction) return cloneElement(child, { size })
+      if (child.type === AlertSecondaryAction) return cloneElement(child, { size })
+
+      if (child.props?.children) {
+        return cloneElement(child, {
+          ...child.props,
+          children: Children.map(child.props.children, (nestedChild: React.ReactElement) =>
+            cloneActions(nestedChild)
+          ),
+        })
+      }
+
+      return child
+    }
+
+    const actions = React.isValidElement(cta) ? cloneActions(cta) : cta
+
+    const layout: { bottom: BoxProps; right: BoxProps } = {
+      bottom: {
+        alignItems: 'flex-start',
+        flexDirection: 'column',
+      },
+      right: {
+        alignItems: { _: 'flex-start', md: 'center' },
+        flexDirection: { _: 'column', md: 'row' },
+      },
+    }
```

Comment:
Can we move to top before the component ?

Reply:
You're right 👍 Fixed here : b3e2450.